packages/pretty-format/src/__tests__/expect-util.js

```diff
@@ -13,41 +13,42 @@ const diff = require('jest-diff');
 const prettyFormat = require('../');
 
 module.exports = {
-  getPrettyPrint: plugins => (received, expected, opts) => {
-    const prettyFormatted = prettyFormat(
-      received,
-      Object.assign(
-        {
-          plugins,
-        },
-        opts,
-      ),
-    );
-    const pass = prettyFormatted === expected;
+  getPrettyPrint: plugins =>
+    function(received, expected, opts) {
```

Comment:
Hade to make this `function` instead of arrow, otherwise I get 
```
  ● HTMLElement Plugin › supports siblings

    TypeError: Cannot read property 'matcherHint' of undefined

      at pass (packages/pretty-format/src/__tests__/expect-util.js:42:20)
      at process._tickCallback (internal/process/next_tick.js:109:7)
```

Rest of the changes is prettier auto-formatting